tenders/hvt/hvt_dumpcore_openbsd_x86_64.c

```diff
@@ -21,21 +21,96 @@
 /*
  * hvt_dumpcore_openbsd_x86_64.c: Glue between the dumpcore module and OpenBSD
  * (x86_64).
+ *
+ * Please note the core generated on OpenBSD is not readable by gdb on OpenBSD.
```

Comment:
What is needed to read a core on OpenBSD?  Using `gdb` from ports?  This should imho be documented.

Reply:
My initial comment stated

> Using gdb from base or ports does not work, the src, core and hvt binary need to be copied to linux for post-mortem debugging

Do you think the above comment should be updated to something like
`Please note the core generated on OpenBSD is not readable by gdb on OpenBSD.  The src, core and hvt binary need to be copied to linux for post-mortem debugging`